### PR TITLE
Added Slovak translation

### PR DESCRIPTION
**Pull request by jakubhi**
Added the Slovak translations of countries, emails and settings (basically everything).

## What does this PR do?

(Translates text into Slovak language)

## Test Plan

()

## Related PRs and Issues

(https://github.com/appwrite/appwrite/issues/1760.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Yes.)

### CODE REVIEW

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
@Meldiron  Might need you to verify this once :+1:

File: app/config/locale/translations/sk.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    "emails.magicSession.signature": "{{project}} tím",
+    "emails.recovery.subject": "Obnovenie hesla",
+    "emails.recovery.hello": "Ahoj {{name}}",
+    "emails.recovery.body": "Použi tento link pre obnovenie svojho {{project}} hesla.",
+    "emails.recovery.footer": "Ak si nepožiadal o obnovu svojho hesla, túto správu môžeš ignorovať.",
+    "emails.recovery.thanks": "Ďakujeme",
+    "emails.recovery.signature": "{{project}} tím",
+    "emails.invitation.subject": "Pozvánka do %s Tímu v %s",
+    "emails.invitation.hello": "Ahoj",
+    "emails.invitation.body": "Tento email ti bol zaslaný, pretože {{owner}} ťa chce pozvať aby si sa stal členom {{team}} tímu v {{project}}.",
```
Comment (Meldiron) [inline app/config/locale/translations/sk.json:26]: I would go with:
```
... zaslaný, pretože {{owner}} ťa pozval, aby si sa stal členom {{team}} tímu v projekte {{project}}.",
```

**Review by Meldiron — CHANGES_REQUESTED** [review]
Great translation 💪 I suggested one change on `emails.invitation.body`.

File: app/config/locale/translations/sk.json (hunk 1 above)
Comment (jakubhi) [reply app/config/locale/translations/sk.json:26]: Yeah, agreed, that sounds better. Should I make a new pull request for this?

**Pull request merged**